### PR TITLE
Permissions cleanup 3

### PR DESCRIPTION
**Pull request by czpython**
screen shot 2016-08-24 at 16 42 17

### CODE REVIEW

File: cms/templates/admin/cms/page/tree/actions_dropdown.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <li
+        {% if has_change_permission %}
+            class="js-cms-tree-item-copy"
+            data-id="{{ page.pk }}"
+            data-apphook="{{ page.application_urls|default_if_none:'' }}"
+        {% endif %}
+    >
+            <a{% if not has_change_permission %} class="cms-pagetree-dropdown-item-disabled"{% endif %} href="#" title="{% filter escapejs %}{% trans 'Copy' %}{% endfilter %}">
+                <span class="cms-icon cms-icon-copy"></span>
+                <span>{% filter escapejs %}{% trans 'Copy' %}{% endfilter %}<span>
```
Comment (FinalAngel) [inline cms/templates/admin/cms/page/tree/actions_dropdown.html]: mixed quotes and double quotes in this template. use double quotes

File: cms/templates/admin/cms/page/tree/menu.html [hunk 2]
```diff
@@ -13,40 +12,42 @@
...
             <div class="cms-tree-item-wrapper">
                 {% if page.is_home %}
                     <div class="cms-tree-item cms-tree-item-icons">
                         <div class="cms-tree-item-inner">
-                                <div class="cms-hover-tooltip cms-hover-tooltip-left cms-hover-tooltip-delay"
-                                    data-cms-tooltip="{% filter escapejs %}{% trans 'home' %}{% endfilter %}">
-                                    <span class="cms-icon cms-icon-home"></span>
-                                </div>
+                            <div class="cms-hover-tooltip cms-hover-tooltip-left cms-hover-tooltip-delay"
+                                data-cms-tooltip="{% filter escapejs %}{% trans 'home' %}{% endfilter %}">
```
Comment (FinalAngel) [inline cms/templates/admin/cms/page/tree/menu.html]: same here, also should be "Home"

**Comment by FinalAngel** [comment]
FE LGTM

**Comment by stefanfoulis** [comment]
LGTM

**Comment by czpython** [comment]
Merged in 737474226075328756ba2d9f8a58a4049b867525 7a54f8d5ae52c09fb72805ad61f190ce5d93dd0a a3a1b33ed3df0cecca254900fcfa1ca39fe61564

**Pull request closed without merging**